docs/userguide/text.md

```diff
@@ -1,144 +1,151 @@
-
-Text styles (size, color, font, ...) are defined by ``TextStyle`` instances. The constructor takes the following parameters:
-
-* ``font`` - name of the font
-* ``size`` - size of the font
-* ``align`` - align of the text, possible values: ``"left"``, ``"middle"``, ``"right"``.
-* ``line_spacing`` - Spacing between lines, specified relatively to ``size`` argument.
-* ``color`` - a string defining the color name
-* ``bold`` - a boolean value if text is bold
-* ``itelic`` - a boolean value if text is italic
-* ``variant_numeric`` - allowed values: ``"normal"``,
-        ``"ordinal"``,
-        ``"slashed-zero"``,
-        ``"lining-nums"``,
-        ``"oldstyle-nums"``,
-        ``"proportinal-nums"``,
-        ``"tabular-nums"``,
-        ``"diagonal-fractions"``,
-        ``"stacked-fractions"``. For meaning see SVG standard.
-
-Each parameter may also be ``None`` (default value) that
-means that the value should be inherited from the parent style.
-
-The simplest use cases, is passing style directly as the second argument of ``tex`` method:
-
-```python
-@elsie.slide()
-def text1(slide):
-    slide.text("Hello world!", elsie.TextStyle(size=70, color="red"))
+You can render text by calling the [`text`](elsie.boxmixin.BoxMixin.text) method on a box.
+By default, it will create a box item that will draw the specified text in the middle of the box:
+```elsie,width=200,height=200
+slide.box().text("Hello world!")
 ```
 
-<img width="512px" height="384px" src="slide_imgs/text1.png">
-
-Each box also inherits from its parent a list of named styles. A style can be named and then referred by its name. The two following examples produce the same slide as the example above:
+## Text style
+*Elsie* draws text using SVG, so you can use common SVG attributes (such as size, color, font) to
+modify the appearance of the rendered text. The style of a text is defined by a
+[`TextStyle`](elsie.textstyle.TextStyle) object. Its constructor takes the following parameters:
+
+- `font`: Name of the used font.
+- `size`: Size of the used font.
+- `align`: Alignment of the text. Allowed values are `"left"`, `"middle"` or `"right"`.
+- `line_spacing`: Spacing between lines. The value is relative to the `size` value.
+- `color`: Color of the text.
+- `bold`: Make the text bold.
+- `itelic`: Make the text italic.
+- `variant_numeric`: Specifies the style of rendering numbers. Allowed values are `"normal"`,
+        `"ordinal"`,
+        `"slashed-zero"`,
+        `"lining-nums"`,
+        `"oldstyle-nums"`,
+        `"proportinal-nums"`,
+        `"tabular-nums"`,
+        `"diagonal-fractions"` or
+        `"stacked-fractions"`.
+See [SVG documentation](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/font-variant)
+for its meaning.
+
+Each parameter may also be `None` (the default value). This means that the value will be inherited
+from its parent style (see below).
+
+If you want to font or text properties that are not available in `TextStyle`, please
```

Comment:
extra "to"

Reply:
Actually it was missing `use` :D But good catch, thanks.